keystone/verify-keystone.sh

```diff
@@ -5,7 +5,148 @@ source /root/openrc
 
 for i in {1..5}; do echo "Test #$i"; openstack user list && break || sleep 5; done
 
-openstack endpoint list
+openstack token issue
 
-openstack project list
 
+#ec2 section
+
+function createEC2 () {
+	openstack ec2 credentials create --user admin
+}
+
+function showNewEC2() {
+	ID=$(openstack ec2 credentials list | awk '!/Access/ && !/+/ {print $2}')
+	Exist=$(openstack ec2 credentials show $ID | awk ' /access/ {print $2}')
+	[[ "$Exist" ]]
+}
+
+function deleteEC2() {
+	ID=$(openstack ec2 credentials list | awk '!/Access/ && !/+/ {print $2}')
+	openstack ec2 credentials delete $ID
+	 Exist=$(openstack ec2 credentials list | grep $ID)
+	[[ ! "$Exist" ]]
+}
+
+#endpoint section
+
+function createEndPoint () {
+	openstack endpoint create identity public http://fakehost:5500/v3 --region RegionOne
+	}
+	
+function showEndPoint () {
+	ID=$(openstack endpoint list | awk '/controller/ {print $2}')
+	[[ "$ID" ]]	
+}
+
+function deleteEndPoint () {
+	ID=$(openstack endpoint list | awk '/controller/ {print $2}')
+	openstack endpoint delete $ID
```

Comment:
this commands seems like failing:
https://travis-ci.org/clearlinux/dockerfiles/builds/181765423#L587